procurement_purchase_requisition_generation/models/stock_rule.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from collections import defaultdict
+
+from odoo import SUPERUSER_ID, api, models
+from odoo.tools import config
+
+
+class StockRule(models.Model):
+    _inherit = "stock.rule"
+
+    def _make_pr_get_domain(self, company_id, values):
+        group = values.get("group_id") or False
```

Comment:
No need of the other part, as by default it returns `None`, which is a falsy value. 

```suggestion
        group = values.get("group_id")
```